config/enums.cwt

```diff
@@ -52,4 +52,11 @@ enums = {
         very_high_ai_values_divergence #  350 #About 85% (46-98%) of characters will be at or below this value for the avarage character
         #Percentages in parentheses are the observed spans
     }
+    enum[attributes] = {
```

Comment:
These are already referenced as `enum[event_option_skills]` around L100 in events.cwt. This enum is never defined anywhere though. I defined event_option_skills in a PR I'm working on right now, but I can modify it to attributes which makes more sense before I post.

Eitherway, I'd reccomend either changing the enum name to event_option_skills, or changing the relevant line in events.cwt. I'll work around whatever you decide.

Reply:
Done in "More documentation, int_value_field and attributes as an enum in events".